src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -21,29 +24,82 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
         const response = await api<ModelCalibrateMutation, ModelCalibrateMutation>(context)
             .withSuccess(ModelCalibrateMutation.ModelCalibrateOptionsFetched)
             .ignoreErrors()
-            .get<DynamicFormMeta>("/model/calibrate/options/");
+            .get<DynamicFormMeta>("model/calibrate/options/");
```

Comment:
Is the extra slash just good practice or was it needed to run the code?

Reply:
I think this was needed - bug in previous branch